capsule_cabinet.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# A script to generate cabinets for fwupd / LVFS
+
+set -e
+
+function die() {
+    echo error: "$@" 1>&2
+    exit 1
+}
+
+if [ ! -f .config ]; then
+    die "no '.config' file in current directory"
+fi
+
+# import coreboot's config file replacing $(...) with ${...}
+while read -r line; do
+    if ! eval "$line"; then
+        die "failed to source '.config'"
+    fi
+done <<< "$(sed 's/\$(\([^)]\+\))/${\1}/g' .config)"
+
+if [ "$CONFIG_DRIVERS_EFI_UPDATE_CAPSULES" != y ]; then
+    die "current board configuration lacks support of update capsules"
+fi
+
+if [ -z $1 ]; then
+    die "No input capsule specified"
+fi
+
+if [ ! -f $1 ]; then
+    die "File $1 not found"
+fi
```

Comment:
Arguments should be checked first before looking for `.config` and sourcing it.  Can also add `$# -eq 1` check to not silently ignore extra arguments.

Reply:
Done in https://github.com/Dasharo/coreboot/pull/655/commits/75ee0b992d961484f6a84b8b54ad279782a0fe4c